### ISSUE TITLE
Automatically replace video links from Twitter

### ISSUE THREAD

**Issue by sdasda7777** [fit-ctu-discord/honza-botner#401]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when embedded twitter videos can't be played and have to be open on new tab. It would be great if we had a bot that could automatically replace links to twitter with links to 3rd party service that allows embedded playing of the videos. (For example fxtwitter.com)

**Describe the solution you'd like**
There probably should be a way to post unaltered twitter links as a failsafe, for example by putting the link in double quotes, or escaping it in some other way. Possibly the bot could leave a reaction on the altered message, to make it clear it has been altered, clicking on which could even restore the original message. Otherwise I think it should be pretty much straightforward?

**Describe alternatives you've considered**
Alternative could be just posting the altered link as a new message, but that could flood the discussions very fast, so I don't think it is optimal.

**Additional context**
![image](https://user-images.githubusercontent.com/17746796/192495906-89d04679-f6e1-40bc-83cf-d9b5bcbbef2e.png)

![image](https://user-images.githubusercontent.com/17746796/192496542-96fa744b-d564-4ade-bac5-0194c9494718.png)

**Comment by sdasda7777** [comment]
Oh, nevermind, it seems editing other people's messages it not something you can do.

**Comment by tenhobi** [comment]
Yep, you cannot edit other people's messages. We could probably hide the embed for all using bot and then place the fixed link? dunno.

**Comment by jirkavrba** [comment]
Imo an ideal solution would be remove the twitter embed from the original message (that should be possible with manage messages permission) and the replying with a fxtwitter link below
